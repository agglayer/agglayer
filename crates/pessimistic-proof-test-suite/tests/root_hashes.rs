use pessimistic_proof::{local_balance_tree::LocalBalanceTree, nullifier_tree::NullifierTree};

#[rstest::rstest]
fn root_hashes_sanity_check() {
    insta::assert_debug_snapshot!(LocalBalanceTree::new().root);
    insta::assert_debug_snapshot!(NullifierTree::new().root);
}
