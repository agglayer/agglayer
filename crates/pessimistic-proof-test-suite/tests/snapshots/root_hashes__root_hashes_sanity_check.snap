---
source: crates/pessimistic-proof-test-suite/tests/root_hashes.rs
expression: "LocalBalanceTree::new().root"
snapshot_kind: text
---
b89931f7384aeddb5c136a679d54464007e2d828d4741bec626ff92aeb4b12d4
